portal-web/test/functional/com/liferay/portalweb/macros/Form.macro

```diff
@@ -301,6 +301,20 @@ definition {
 			value1 = "${formPageName}");
 	}
 
+	macro editPageDescription {
+		var key_titleNumber = "${formTitleNumber}";
```

Comment:
@ewertonluna are you using this `key_titleNumber` in the `PAGE_DESCRIPTION_FIELD` path?